### ISSUE TITLE
Upgrade Microsoft.Extensions.Logging

### ISSUE THREAD

**Issue by darraghjones** [mattwcole/gelf-extensions-logging#70]
We're getting the following error when trying to use this library with a more recent version of `Microsoft.Extensions.Logging`

```
Could not load type 'Microsoft.Extensions.Logging.Abstractions.Internal.NullScope' from assembly 'Microsoft.Extensions.Logging.Abstractions, Version=3.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60
```
I believe the issue is to do with this code https://github.com/mattwcole/gelf-extensions-logging/blob/dev/src/Gelf.Extensions.Logging/GelfLogger.cs#L7 and the fact that the NullScope has been moved and/or made internal.

**Comment by mattwcole** [comment]
Hi, thanks for reporting. You're right it looks like `NullScope` was made internal recently. I have published a fix in [v2.5.0-pre2](https://www.nuget.org/packages/Gelf.Extensions.Logging/2.5.0-pre2). Stable version to come in a few days.

**Comment by mattwcole** [comment]
Stable version [v2.5.0](https://www.nuget.org/packages/Gelf.Extensions.Logging/2.5.0) released.
